ui/admin/app/components/worker-diagram/dual-filter/index.hbs

```diff
@@ -37,7 +37,11 @@
         {{t 'resources.target.workers.diagram.client'}}
       </D.Item>
       <D.Item @icon='settings'>
-        {{t 'resources.target.workers.diagram.front-line-worker'}}
+        {{#if (feature-flag 'target-worker-filters-v2-hcp')}}
+          {{t 'resources.target.workers.diagram.hcp-worker'}}
+        {{else}}
+          {{t 'resources.target.workers.diagram.front-line-worker'}}
+        {{/if}}
```

Comment:
It was suggested that we split this component into 2 new ones. `WorkerDiagram::DualFilter` and `WorkerDiagram::DualFilter::Hcp`. The only difference would be the logic here and on lines 64-68. Then we could remove the tests that check for the correct language rendered. We would just need to verify the correct component was rendered as the language will be unique to that component.